tests/core/test_message.c

```diff
@@ -268,7 +269,8 @@ void test_message_with_tagged_data_serialize() {
 
   // create tagged data
   byte_t data[] = {0x48, 0x65, 0x6c, 0x6c, 0x6f, 0x20, 0x57, 0x6f, 0x72, 0x6c, 0x64};
-  msg->payload = tagged_data_create("iota.c lib", data, sizeof(data));
+  char const* const iotac_lib = "iota.c lib";
+  msg->payload = tagged_data_create((byte_t*)iotac_lib, strlen(iotac_lib), data, sizeof(data));
```

Comment:
should include the byte of the null terminator

Reply:
Done.